### DISCUSSION TITLE
Is there a way to deliver without creating `Noticed::Notification`

### DISCUSSION THREAD

**Discussion by edwinthinks** [excid3/noticed#500] [Q&A]
This might be a very strange use-case but I wanted to double check if anyone here has ever needed to do this or have suggestions.

We currently have code that ships off emails to recipients via the gem. The left over `Noticed::Notifications` are visible in the list of notifications that we show to our users. Is there a clever way to either delete those, not create those, or make those invisible using the gem?

If not, then I know there are several ways to handle this within our codebase via filtering. I figured it wouldn't hurt to ask the community if there were ideas here.

**Comment by excid3** [comment]
Use the Noticed::Ephemeral class.

```ruby
class MyNotification < Noticed::Ephemeral
  deliver_by :email, ...
end
```
